src/main/groovy/com/blackbaud/templates/project/DatasourceProject.groovy

```diff
@@ -44,7 +44,7 @@ class DatasourceProject {
         applyPostgresCompileDependencies()
         applyPostgresApplicationProperties()
         applyTestCleanupSql()
-        basicProject.appendServiceToAppDescriptor("postgres-shared")
+//        basicProject.appendServiceToAppDescriptor("postgres-shared")
```

Comment:
why include this commented?

Reply:
well, the appendServiceToAppDescriptor method got removed but i haven't yet removed the postgres code generation which i'm going to do in a follow-up PR, was going to remove the line then